extensions/oplogPopulator/modules/ConnectorsManager.js

```diff
@@ -238,47 +222,98 @@ class ConnectorsManager {
         }
     }
 
+    /**
+     * Spawns a connector when buckets are configured for it and is not running,
+     * or destroys connector with no buckets configured
+     * @param {Connector} connector connector instance
+     * @returns {Promise<Boolean>} true if connector state changed
+     * @throws {InternalError}
+     */
+    async _spawnOrDestroyConnector(connector) {
+        try {
+            if (connector.isRunning && connector.bucketCount === 0) {
+                await connector.destroy();
+                this._metricsHandler.onConnectorDestroyed(false);
```

Comment:
there is no param in this function....
and if it were, it could be either true or false depending on how that specific connector was created...